<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnTakeExam">
       <property name="text">
        <string>Kerjakan Ujian</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddTask">
       <property name="text">
        <string>Tambah Ujian</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnViewResults">
       <property name="text">
        <string>Lihat Hasil Ujian</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddQuestion">
       <property name="text">
        <string>Tambah Soal</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>155</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Pencarian :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Filter Role:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="searchLineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Cari</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="roleComboBox"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Data User</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ujian</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Hasil</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
